cookbook-react/.stylelintrc.js

```diff
@@ -9,5 +9,6 @@ module.exports = {
       ignoreInsideAtRules: ['if', 'mixin', 'media']
     }],
     'scss/at-import-no-partial-leading-underscore': null,
+    'plugin/stylelint-no-indistinguishable-colors': null
```

Comment:
I don't think removing this from the cookbook is a good idea if we keep it in projects.
We can tune the rule or consider removing it, but it's useful for accessibility.